lambdas/sanctions-check.ts

```diff
@@ -23,30 +23,47 @@ export const handler = async (event: any = {}): Promise<any> => {
 
   const request = typeof event.body == 'object' ? event.body : JSON.parse(event.body);
 
-  const sanctionChecks = Array.isArray(request) ? request : [request];
-
-  for (const check of sanctionChecks) {
-    const address = check.address;
-    if (!address) {
-      return formatResponse(400, 'Error: You are missing the address in one of your sanction checks');
-    }
+  if (!request.address) {
```

Comment:
I think we only ever checked one address at a time.. so is this fine?

Reply:
yea that's true. i only added arrays to be consistent with TRM.